test/units/utils/test_encrypt.py

```diff
@@ -35,31 +21,12 @@ def assert_hash(expected, secret, algorithm, **settings):
         assert excinfo.value.args[0] == "passlib must be installed and usable to hash with '%s'" % algorithm
```

Comment:
@mkrizek the entire else-branch here lost coverage with the changes in this PR. Should the assertion here be made unconditional or new tests added?

Reply:
See https://app.codecov.io/gh/ansible/ansible/commit/6935c8e303440addd3871ecf8e04bde61080b032/indirect-changes#3ab5fa106f6a24247d92fd39c8e79a4f-R19

Reply:
And this is the only place with dead code in this test module: https://app.codecov.io/gh/ansible/ansible/blob/devel/test%2Funits%2Futils%2Ftest_encrypt.py